db/migrate/20160217192112_create_rentals.rb

```diff
@@ -0,0 +1,14 @@
+class CreateRentals < ActiveRecord::Migration
+  def change
+    create_table :rentals do |t|
+      t.string :rental_status
+      t.integer :user_id, null: false
+      t.integer :department_id, null: false
+      t.integer :reservation_id, null: false
+      t.integer :fee_schedule_id, null: false
+      t.datetime :checked_out_at
+      t.datetime :checked_in_at
```

Comment:
Are `checked_out_at` and `checked_in_at` still needed with the statemachine setup?